src/server/core/handler.rs

```diff
@@ -169,16 +180,29 @@ impl<C: Connection, H: server::Handler<C>> server::Handler<C> for Handler<H> {
         }
     }
 
-    fn handle_property<'c>(&self, name: &str, requested_value: Option<&[u8]>, conn: &mut C, send_buffer: &mut [u8]) -> Option<usize> {
+    fn handle_sub<'c>(&self, name: &str, conn: &mut C, send_buffer: &mut [u8]) -> Option<usize> {
+        use common::core::msg::prerecorded::publish_property;
+        //FIXME we do not support subscribing to properties yet, so just return
+        //the current value
+        if name == "core.server-msg-bytes-max" {
+            publish_property(send_buffer, name, &conn.max_server_message_length())
+        } else if name == "core.client-msg-bytes-max" {
+            publish_property(send_buffer, name, &conn.max_client_message_length())
+        } else {
+            self.next.handle_sub(name, conn, send_buffer)
+        }
+    }
+
+    fn handle_set<'c>(&self, name: &str, requested_value: &[u8], conn: &mut C, send_buffer: &mut [u8]) -> Option<usize> {
         use common::core::msg::prerecorded::publish_property;
-        //we do not support changing any properties yet, so just return the
-        //current value
+        //FIXME we do not support changing any properties yet, so just return
```

Comment:
That's not necessarily a FIXME. It just refers to us not having any connection-handling code that affords changing buffer sizes.